leases/app/lib/LeaseStore.scala

```diff
@@ -37,8 +39,8 @@ object LeaseStore {
   private def uuid = Some(UUID.randomUUID().toString)
   private def key(id: String) = UniqueKey(KeyEquals('id, id))
 
-  def put(lease: MediaLease) = Scanamo.put(client)(tableName)(lease.copy(id=uuid))
-  def delete(id: String) = Scanamo.delete(client)(tableName)(key(id))
+  def put(lease: MediaLease) = Future { Scanamo.put[MediaLease](client)(tableName)(lease.copy(id=uuid))  }
+  def delete(id: String) = Future { Scanamo.delete(client)(tableName)(key(id)) }
```

Comment:
I added these so I could use the results as promises (i.e. chain subsequent actions), is this the right way to do it? 

I noticed the other methods in here returned mappable objects but these didn't so I wasn't sure.